apps/displays/Sensics_dSight_portrait_1input_sbs.json

```diff
@@ -43,7 +43,7 @@
       {
         "center_proj_x": 0.5,
         "center_proj_y": 0.5,
-        "rotate_180": 0
+        "rotate_180": 180
```

Comment:
this is a bool masquerading as an int - don't recall exactly but pretty sure either 1 or true will do what you want. I know 1 will do what you want. I suspect 180 will not.